packages/react-resizable-panels-website/tests/CursorStyle.spec.ts

```diff
@@ -1,12 +1,36 @@
 import { expect, Page, test } from "@playwright/test";
 import { createElement } from "react";
 import { Panel, PanelGroup, PanelResizeHandle } from "react-resizable-panels";
-import { getCursorStyle } from "react-resizable-panels/src/utils/cursor";
```

Comment:
This was reaching for a private~ util (and with `exports` we can't reach into undeclared files, although we could add `src/*` "forwarding", I just wanted to avoid this). I decided to inline it in this test file - if you have any other preferred about to solve this, let me know.